lib/routes/user.js

```diff
@@ -5,36 +5,39 @@ var cors = require('cors');
 var userService = require('../services/user-service');
 var statusObj = require("../status");
 
-function userRoute(){
+function userRoute() {
   var user = new express.Router();
   user.use(cors());
   user.use(bodyParser());
 
-  user.get('/', function(req, res){
+  user.get('/', function (req, res) {
     res.json(userService.getRandomUser());
   });
 
-  user.get('/generate', function(req, res){
-    if(statusObj.isFinished()){
+  user.get('/generate', function (req, res) {
+    if (statusObj.isFinished()) {
+      res.statusCode = 200;
       res.json(statusObj);
       statusObj.clearStatus();
-    }else if(statusObj.isInProgress()){
+    } else if (statusObj.isInProgress()) {
+      res.statusCode = 255;
       res.json(statusObj);
-    }else{
+    } else {
       insertDataIntoDb(req.query.numberOfRecords);
+      res.statusCode = 255;
```

Comment:
255 is invalid status code (I do not know that status code). 
I think that proper status core here will be 202.